src/eve-server/system/DestinyManager.cpp

```diff
@@ -3350,7 +3350,9 @@ void DestinyManager::SendDestinyUpdate( std::vector<PyTuple*>& updates, std::vec
         mySE->SystemMgr()->GetClientList(cv);
 
         for(auto const& value: cv) {
-            value->GetShipSE()->SysBubble()->BubblecastDestiny(updates, events, "destiny");
+             if (value->GetShipSE() != nullptr) {
```

Comment:
Not sure how this is ever null.  "Normal" stations this works just fine but any outpost or even cloned station (manually editing the database) causes the ShipSE to be null upon docking.